src/Concerns/ManagesSessionPool.php

```diff
@@ -105,27 +103,22 @@ public function __debugInfo()
         // HACK: Use reflection to extract some information from a private method
         // -------------------------------------------------------------------------
         $session = null;
-        /** @var FetchAuthTokenInterface|null $credentialFetcher */
         $credentialFetcher = null;
+
         $internalConnectionProperty = (new ReflectionObject($this->getSpannerClient()))->getProperty('connection');
-        if ($internalConnectionProperty !== null) {
-            $internalConnectionProperty->setAccessible(true);
-            /** @var ConnectionInterface $internalConnection */
```

Comment:
Unnecessary. Type is already set below using `$internalConnection instanceof Grpc`.